### DISCUSSION TITLE
### Issue in Setting Up Sunbird-RC Locally Using Docker

### DISCUSSION THREAD

**Discussion by VISHNUDAS-tunerlabs** [Sunbird-RC/community#1012] [Issues] [answered]
**Problem:**
We are trying to set up the Sunbird-RC service locally following the [documentation](https://rc.sunbird.org/use/getting-started/installation-guide/manual-installation-through-docker-compose). After pulling the images and attempting to run the services, we encounter the following error:

```
sunbird-rc-core-vault-1 Error
```

**Environment Configuration:**
Here is the environment configuration we have set for Vault, which we obtained by following the documentation:

```env
RELEASE_VERSION=v2.0.0

# vault service
VAULT_ADDR=http://0.0.0.0:8200
VAULT_API_ADDR=http://0.0.0.0:8200
VAULT_ADDRESS=http://0.0.0.0:8200

# identity service
VAULT_ADDR=http://vault:8200
VAULT_TOKEN=aass
VAULT_BASE_URL=http://vault:8200/v1
VAULT_ROOT_PATH=http://vault:8200/v1/kv
VAULT_TIMEOUT=5000
VAULT_PROXY=false
SIGNING_ALGORITHM=Ed25519Signature2020
JWKS_URI=
ENABLE_AUTH=false
WEB_DID_BASE_URL=https://example.com/identifier
```

**Request for Assistance:**
Can anyone assist us in identifying what we might be doing wrong? Any guidance or suggestions would be greatly appreciated.

Thank you!

---

cc- @tejash-jl

**Comment by AmiableAnil** [comment]
@surendrasinghs can you please tag the right person to look into this issue.

**Comment by holashchand** [comment, marked as answer]
Hi @VISHNUDAS-tunerlabs , @AmiableAnil 
we need to setup the vault first before starting any other services.
Here is the guide for it
https://github.com/Sunbird-RC/devops/tree/main/deploy-as-code/docker/v2/registry_and_credentialling

**Comment by VISHNUDAS-tunerlabs** [comment]
Hi Holash,
We will check this and get back to you
Thank you
Vishnu
